raiden/tests/utils/transfer.py

```diff
@@ -1031,3 +1035,21 @@ def make_receive_expired_lock(
     )
 
     return receive_lockedtransfer
+
+
+def calculate_amount_to_drain_channel(deposit: int) -> int:
+    # calculate the amount to send so that including fees it covers the entire deposit
+    denominator = (
+        1
+        + INTERNAL_ROUTING_DEFAULT_FEE_PERC
+        + INTERNAL_ROUTING_DEFAULT_FEE_PERC * DEFAULT_MEDIATION_FEE_MARGIN
+    )
```

Comment:
For me it would be a bit easier to follow as
```
    1 + INTERNAL_ROUTING_DEFAULT_FEE_PERC * (1 + DEFAULT_MEDIATION_FEE_MARGIN)
```
Same for `calculate_fee_for_amount` below.

Reply:
For me not so much but if it helps you I am changing it! I want my code to make sense to colleagues.